medicine-and-science-in-sports-and-exercise.csl

```diff
@@ -66,27 +71,58 @@
     </choose>
   </macro>
   <macro name="access">
-    <group delimiter=" ">
-      <group prefix="[" suffix="]" delimiter=" ">
-        <text term="cited"/>
-        <date variable="accessed" suffix=" ">
-          <date-part name="year"/>
-          <date-part name="month" prefix=" " form="short" strip-periods="true"/>
-          <date-part name="day" prefix=" "/>
-        </date>
-      </group>
-      <group>
-        <text value="Available from: "/>
-        <text variable="URL"/>
-      </group>
-    </group>
+    <choose>
+      <if type="article-journal" match="any">
```

Comment:
I'm afraid this doesn't quite work: If you look at
Vickers AJ. Time course of muscle soreness following different types of exercise. BMC Musculoskeletal Disorders [Internet]. 2001 [cited 2001 May 31];2(5). Available from: http://www.biomedcentral.com/1471-2474/2/5. doi:10.1186/1471-2474-2-5.

The volume and issue information is between access date and URL/DOI. The way you'd have to do this is to the [cited 2001 May 3] onto the date macro and then have only URL/DOI in access all the way at the end.